tracer/src/Datadog.Trace/Activity/Handlers/IgnoreActivityHandler.cs

```diff
@@ -25,6 +25,9 @@ internal class IgnoreActivityHandler : IActivityHandler
             "Npgsql",
             "System.Net.Http.Desktop",
             "SqlClientDiagnosticListener",
+            "Experimental.System.Net.NameResolution",
```

Comment:
Maybe it'd make sense to add some globbing support here to ignore `Experimental.System.*` ones

not for this PR, I don't think this has glob support

Reply:
Yeah, good idea 🤔 We might well already have "prefix" support. We can always update this in a follow up PR anyway 🙂